gnomad_qc/v4/create_release/update_hgdp_tgp_subset_AF.py

```diff
@@ -0,0 +1,580 @@
+    logger.info("Calculating AFs for selected samples...")
+    freq_ht_all = calculate_callstats_for_selected_samples(
+        mt,
+        samples_ht,
+    )
+
+    if samples_ht.filter(samples_ht.hgdp_tgp_meta.project == "HGDP").count() > 0:
+        freq_ht_hgdp = calculate_callstats_for_selected_samples(
+            mt,
+            samples_ht.filter(samples_ht.hgdp_tgp_meta.project == "HGDP"),
+            subsets=["hgdp"],
+        )
+    else:
+        freq_ht_hgdp = None
+
+    if (
+        samples_ht.filter(samples_ht.hgdp_tgp_meta.project == "1000 Genomes").count()
+        > 0
+    ):
+        freq_ht_tgp = calculate_callstats_for_selected_samples(
+            mt,
+            samples_ht.filter(samples_ht.hgdp_tgp_meta.project == "1000 Genomes"),
+            subsets=["tgp"],
+        )
+    else:
+        freq_ht_tgp = None
+
+    logger.info("Concatenating AFs for selected samples...")
+    subset_freq_hts = []
+    if freq_ht_hgdp is not None:
+        subset_freq_hts.append(freq_ht_hgdp.select("freq").select_globals("freq_meta"))
+    if freq_ht_tgp is not None:
+        subset_freq_hts.append(freq_ht_tgp.select("freq").select_globals("freq_meta"))
+
+    freq_ht = concatenate_subset_frequencies(freq_ht_all, subset_freq_hts)
+
+    return freq_ht
+
+
+def remove_pops_from_freq_meta(ht: hl.Table, pops_to_remove: List[str]) -> hl.Table:
+    """
+    Remove populations from the freq_meta.
+
+    :param ht: Table with call stats.
+    :param pops_to_remove: List of populations to be removed.
+    :return: Table with the populations removed.
+    """
+    pops_to_remove = hl.literal(pops_to_remove)
+    ht = ht.annotate_globals(
+        freq_meta=ht.freq_meta.map(
+            lambda d: d.filter(lambda x: ~pops_to_remove.contains(x))
+        ),
+        freq_index_dict=hl.dict(
+            hl.zip(
+                ht.freq_index_dict.keys().filter(lambda k: ~pops_to_remove.contains(k)),
+                ht.freq_index_dict.values(),
+            )
+        ),
+    )
+    return ht
+
+
+def main(args):
+    """
+    Script to update call stats for HGDP + 1KG subset for v4.
+
+    .. note::
+    This code is specifically designed for the update HGDP + 1KG subset in
+    v4 release HT. There are a few major changes compared to the v3 release:
+        - the following new sample filters are applied: hard filters,
+        pop PC outliers,relatedness within the subset and relatedness to
+        the rest of the release
+        - the new pop labels
+        - the new split of the `Han` and `Papuan` samples
+    In order to avoid re-calculating the callstats for the whole subset / whole
+    release, we will calculate the callstats for the samples that will be added
+    and subtracted, then merge the callstats with the old callstats in the
+    release HT.
+    """
+    test = args.test
+
+    hl.init(
+        log="/update_hgdp_tgp_af.log",
+        default_reference="GRCh38",
+        tmp_dir="gs://gnomad-tmp-4day",
+    )
+
```

Comment:
Use resource function I added

```suggestion
    v4_genome_release_resources = get_v4_genomes_release_resources(
        test=test, overwrite=overwrite
    )
```